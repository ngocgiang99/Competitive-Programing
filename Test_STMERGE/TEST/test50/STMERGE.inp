4
10 1
342143470 
935896768 
324861669 
830920686 
250209118 
326382011 
82494437 
774135974 
383894153 
192557040 
7 1
556125956 
16455555 
680148239 
567373989 
481993379 
311243994 
949011172 
9 3
702236313 942799125 8793885 
137671138 222518694 335567382 
244048091 43286636 773615910 
688797854 439169846 432978414 
923878013 998445985 314781619 
795350910 367910737 647209555 
542734871 809594480 352444295 
179889706 72525927 435774554 
594268995 404972251 178534633 
2 7
38593116 564027424 882823847 469587235 963753184 942292179 844358005 
28510930 416442076 699162118 871168014 21936922 843077383 110615806 
