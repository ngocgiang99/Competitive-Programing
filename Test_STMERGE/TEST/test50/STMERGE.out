192557040
556125956
8793885
28510930
